src/lib/AccountUtils/deserialize.ts

```diff
@@ -12,9 +12,26 @@ import {
 import { WalletType } from 'types/wallet';
 import { stringToPath } from '@cosmjs/crypto/build/slip10';
 import { fromHex } from '@cosmjs/encoding';
+import deviceInfoModule from 'react-native-device-info';
 
 const ACCOUNT_ALGOS = ['secp256k1', 'ed25519', 'sr25519'];
 
+/**
+ * Function to extract the account creation time from the serialized account.
+ * NOTE: The creationTime field has been added after the version 2.0 of DPM
+ * so there can be accounts without the creationTime field in such cases
+ * to approximate the account creation time will be used the application
+ * installation time.
+ * @param account - Account from which the date will be extracted.
+ */
+const deserializeCreationTime = (account: Partial<SerializableAccount>): Date => {
```

Comment:
@RiccardoM @bragaz Since we want to track when an account has been created/imported i have added this function to fallback to to the application installation date to aproximate the timestamp of when an account has been imported/created before the implementation of PostHog.